src/docgen/transpile/transpile.ts

```diff
@@ -425,6 +425,15 @@ export class TranspiledTypeReference {
       if (!this.ref.fqn) {
         throw new Error(`Original type reference for ${this.type.fqn} does not have a fqn.`);
       }
+
+      // If we are running in a test, report a fake stable version since types
+      // may belong to dependency packages, which could be specified with caret
+      // dependencies - this means the packageVersion of a type like
+      // `construct.Construct` will be set to whichever version is installed.
+      // This is okay in practice, but makes snapshot tests inconsistent.
```

Comment:
Why is this a problem though? Our snapshots will get updated during self-mutation

Reply:
Scratch that, its inconsistent over time you mean, unrelated to dependency upgrades 👍